subsys/bluetooth/host/smp.c

```diff
@@ -2111,6 +2138,27 @@ static u8_t send_pairing_rsp(struct bt_smp *smp)
 }
 #endif /* CONFIG_BT_PERIPHERAL */
 
+static u8_t smp_pairing_allow_query(
+	struct bt_conn *con,
+	struct bt_smp_pairing *pairing)
+{
+#if defined(BT_SMP_PAIRING_ALLOW)
```

Comment:
`CONFIG_BT_SMP_PAIRING_ALLOW`. What worries me a bit is that this indicates that you never tested the code.